modules/python/init.zsh

```diff
@@ -65,6 +110,12 @@ if (( $+commands[pip] )); then
   source "$cache_file"
 
   unset cache_file
+
+  # Load auto workon cwd hook
+  if zstyle -t ':prezto:module:python' enable-virtualenv-auto-workon-cwd 'yes'; then
```

Comment:
Opinions on the name? Something shorter would be nice. We've used stuff like `:prezto:module:tmux:auto-start` in other places... how would `:prezto:module:python:auto-cwd-virtualenv` be? Suggestions would be welcome because I'm terrible at naming things.